app/workers/copy_project_job.rb

```diff
@@ -77,7 +77,7 @@ def create_project_copy(source_project,
     UserMailer.with_deliveries(send_mails) do
       parent_id = target_project_params[:parent_id]
       target_project = Project.new.tap do |p|
-        p.safe_attributes = target_project_params
+        p.attributes = target_project_params
```

Comment:
@why-el this allows to set more attributes now, than it used to. Please double check, whether this is a problem.